docs/hazmat/primitives/asymmetric/utils.rst

```diff
@@ -28,3 +28,50 @@ Asymmetric Utilities
     :param int s: The raw signature value ``s``.
 
     :return bytes: The encoded signature.
+
+.. function:: encode_ec_point(curve, x, y)
+
+    .. versionadded:: 1.1
+
+    Encodes an elliptic curve point to a byte string as described in
+    `SEC 1 v2.0`_ section 2.3.3. This function only supports uncompressed
+    points.
+
+    :param curve: A
+        :class:`~cryptography.hazmat.primitives.asymmetric.ec.EllipticCurve`
+        provider.
+
+    :param x: The x value of the point.
+
+    :type: int or None
```

Comment:
Not sure what you're trying to do here, but it's not rendering the way you want: https://jenkins.cryptography.io/job/cryptography-pr-docs/826/artifact/cryptography/docs/_build/html/hazmat/primitives/asymmetric/utils.html#cryptography.hazmat.primitives.asymmetric.utils.encode_ec_point